lib/std/compress/flate/inflate.zig

```diff
@@ -342,6 +342,8 @@ pub fn Inflate(comptime container: Container, comptime ReaderType: type) type {
 }
 
 test "flate.Inflate struct sizes" {
+    if (@sizeOf(usize) != 8) return error.SkipZigTest;
```

Comment:
What is the purpose of these struct size tests? If it does not apply to 32 bit systems, it seems like they may not be very valuable tests after all.

In fact, Zig does not guarantee the `@sizeOf` structs, and so these tests are not valid.

Reply:
It was useful for me during implementation to understand where do static allocations come from and to compare memory usage with current implementation. 
I agree that they don't make sense any more. Will remove them.